packages/core/rendering/core-rendering-server-internal/src/views/template.tsx

```diff
@@ -80,9 +80,15 @@ export const Template: FunctionComponent<Props> = ({
             {logo}
             <div
               className="kbnWelcomeText"
-              data-error-message={i18n('core.ui.welcomeErrorMessage', {
+              data-error-message-title={i18n('core.ui.welcomeErrorMessageTitle', {
+                defaultMessage: 'Elastic did not load properly',
+              })}
+              data-error-message-text={i18n('core.ui.welcomeErrorMessageText', {
                 defaultMessage:
-                  'Elastic did not load properly. Check the server output for more information.',
+                  'Please reload this page. If the issue persists, check the server logs.',
```

Comment:
nit: should we mention the browser's console as well?

AFAIK, more often than not, this error is likely an issue loading assets, and it will probably be more obvious to start with the browser's console logs.

Reply:
Getting back to this, how about? 


`Please reload this page. If the issue persists, check the browser console and server logs.`